### PR TITLE
Drop Travis CI

### LINKED ISSUE #691: Drop Travis CI
With https://github.com/playframework/play-json/pull/682 GitHub actions is responsible for CI.
If that works as expected, Travis should be dropped.

### PR DESCRIPTION
**Pull request by ihostage**
Fixes #691

### CODE REVIEW

**Comment by ihostage** [comment]
Build failed by Mergify. But Mergify configuration fixed in this PR 😄

**Review by cchantep — APPROVED** [review]

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       matrix:
         jdk: [ 11, 8 ]
-        scala: [ 2.12.15, 2.13.6, 3.0.0 ]
+        scala: [ 2.12.15, 2.13.7, 3.0.2 ] # Should be sync with Mergify conditions (.mergify.yml)
```
Comment (mkurz) [inline .github/workflows/build-test.yml:84]: Scala 2.13.8 out already (at least on Maven Central)

File: .mergify.yml [hunk 2]
```diff
@@ -1,21 +1,44 @@
...
+      # Conditions to get out of the queue (= merged)
+      - check-success=Check / Code Style
+      - check-success=Check / Binary Compatibility
+      - check-success=Check / Docs
+      - check-success=Check / Tests (Scala 2.12.15 & JDK 11)
+      - check-success=Check / Tests (Scala 2.13.7 & JDK 11)
+      - check-success=Check / Tests (Scala 3.0.2 & JDK 11)
+      - check-success=Check / Tests (Scala 2.12.15 & JDK 8)
+      - check-success=Check / Tests (Scala 2.13.7 & JDK 8)
+      - check-success=Check / Tests (Scala 3.0.2 & JDK 8)
```
Comment (mkurz) [inline .mergify.yml:13]: Hmm.. Would probably be nice if we could catch all with e.g. `check-success=Check / *`

**Comment by mkurz** [comment]
> Build failed by Mergify. But Mergify configuration fixed in this PR smile

Yes that's why I need to merge by hand... Because still the current broken mergify config from the `main` branch is used, not the one you modified in this PR :wink:

File: .mergify.yml (hunk 2 above)
Comment (ihostage) [reply .mergify.yml:13]: @mkurz Unfortunately not 😞 
https://docs.mergify.com/conditions/#validating-all-status-checks

Comment (mkurz) [reply .mergify.yml:13]: :cry:

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (ihostage) [reply .github/workflows/build-test.yml:84]: I tried, but not all dependencies are ready for 2.13.8 🤷‍♂️

Comment (mkurz) [reply .github/workflows/build-test.yml:84]: :smile: Yes I see now in #694

**Pull request merged**